opslevel/terraform_type_conversions.go

```diff
@@ -98,6 +98,14 @@ func ListValueToStringSlice(ctx context.Context, listValue basetypes.ListValue)
 	return dataAsSlice, diags
 }
 
+// Converts a basetypes.ListValue to a []string (returns nil if input list value is null)
+func ListValueToStringSliceOrNil(ctx context.Context, listValue basetypes.ListValue) ([]string, diag.Diagnostics) {
```

Comment:
This can be dropped since this is only used on a field that cannot be null.
```go
	if planModel.OwnershipTagKeys.IsNull() {
		panic("ownershipTagKeys cannot be null")
	}
	ownershipTagKeys, diags := ListValueToStringSliceOrNil(ctx, planModel.OwnershipTagKeys)
```
Running `terraform plan` with this field set to null runs without pancking